spec/lib/tasks/service_spec.rb

```diff
@@ -0,0 +1,49 @@
+describe "Service tasks" do
+  describe "service:duplicate" do
+    it "duplicates the service" do
+      old_service = create(:service, :all_tiers, lgsl_code: 1)
+
+      service_interaction1 = create(:service_interaction, service: old_service)
+      create_list(:link, 3, service_interaction: service_interaction1)
+
+      service_interaction2 = create(:service_interaction, service: old_service)
+      create_list(:link, 3, service_interaction: service_interaction2)
+
+      args = Rake::TaskArguments.new(%i[from_lgsl_code to_lgsl_code], [1, 2])
+      Rake::Task["service:duplicate"].execute(args)
+
+      new_service = Service.find_by!(lgsl_code: 2)
+
+      expect(new_service.label).to eq("Transitioning #{old_service.label}")
+      expect(new_service.slug).to eq("transitioning-#{old_service.slug}")
+
+      expect(new_service.interactions.count).to eq(old_service.interactions.count)
+      expect(new_service.service_interactions.count).to eq(old_service.service_interactions.count)
+      expect(new_service.links.count).to eq(old_service.links.count)
+    end
+  end
+
+  describe "service:rename" do
+    it "should rename the service label and slug" do
+      create(:service, :all_tiers, lgsl_code: 1)
+
+      allow_any_instance_of(Service).to receive(:update!)
+        .with({ label: "Updated label", slug: "updated-slug" })
+        .and_return(true)
+
+      args = Rake::TaskArguments.new(%i[lgsl_code label slug], [1, "Updated label", "updated-slug"])
+      Rake::Task["service:rename"].execute(args)
+    end
```

Comment:
This test doesn't have any expectations and so doesn't actually test anything.

I'd suggest changing it round to something like:

```
service = create(:service, :all_tiers, lgsl_code: 1)
args = Rake::TaskArguments.new(%i[lgsl_code label slug], [1, "Updated label", "updated-slug"])
expect { Rake::Task["service:rename"].execute(args) }
  .to change { service.reload.label }
  .from(
```
etc